custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -0,0 +1,213 @@
+            "mandatory": true,
+            "columnRole": "documents_to_tag",
+            "allowedColumnTypes": [
+                "string"
+            ]
+        },
+        {                                           #TODO add a comment about the supported languages? 
+            "name": "language_column",
+            "label":'Language column',
+            "type": "COLUMN",
+            "mandatory":true,
+            "columnRole": "documents_to_tag",
+            "allowedColumnTypes": [
+                "string"
+            ]
+            
+        },
+        
+        {
+            "name": "tags_dataset",
+            "label": "Tags dataset",
+            "type": "SEPARATOR"
+        },
+        
+        {
+            "name": "tag_column",
+            "label": "Tag column",
+            "type":"COLUMN",
+            "mandatory":true,
+            "columnRole":"ontology_of_tags",
+            "allowedColumnTypes": [
+                "string"
+            ]
+        },
+        
+       
+         
+        {
+            "name":"keyword_column",
+            "label":"Keyword column",
+            "type":"COLUMN",
+            "mandatory":false,
+            "columnRole":"ontology_of_tags",
+            "allowedColumnTypes": [
+                "string"
+            ]
+            
+        },
+        
+        
+        {
+            "name":'category_column',
+            "label":"Category column",
+            "type": "COLUMN",
+            "mandatory":false,
+            "columnRole":"ontology_of_tags",
+            "allowedColumnTypes": [
+                "string"
+            ],
+            "defaultValue":""
+            
+        },
+        
+        {
+            "name": "Matching Parameters",
+            "label": "Matching Parameters",
+            "type": "SEPARATOR"
+        },
+        
+        {
+            "name":'case_sensitivity',
+            "label":"Case-sensitive output",
+            "description": "for keywords compared case sensitive by exact match",
+            "type": "BOOLEAN",
+            "defaultValue":true
+            
+        },
+        {
+            "name":"lemmatization",
+            "label":"Lemmatization",
+            "description": "for matching by lemmatized forms",
+            "type": "BOOLEAN",
+            "defaultValue":false
+            
+        },
+        {
+            "name":'unicode_normalization',
+            "label":"Unicode normalization",
+            "description": "for matching regardless character encoding",
+            "type": "BOOLEAN",
+            "defaultValue":false
+            
+        },
+        
+        
+          {
+            "name": "Output Formats",
+            "label": "Output Formats",
+            "type": "SEPARATOR"
+        },
+        
+
+        {
+            "name": "possible_outputs_with_cat",
+            "label":"Possible outputs",
+            "type": "SELECT",
+            "visibilityCondition":"model.category_column.length>0",
```

Comment:
That's == 0, right?

Reply:
You mean the other condition i suppose, and yes, i'll change it

Reply:
done in 445d2d4